src/backend/views/auth.twig

```diff
@@ -8,6 +8,9 @@
     <p>
       Enter a password to access pages editing
     </p>
+    <p>
+      {{ header }}
```

Comment:
what is it?

Reply:
Alexander made it to see what error occurs